src/ArgentPonyWarcraftClient.Extensions.DependencyInjection/ArgentPonyWarcraftClient.Extensions.DependencyInjection.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <Copyright>Copyright © Dan Jagnow 2017</Copyright>
     <NeutralLanguage>en-US</NeutralLanguage>
-    <Title>Argent Pony Warcraft Client</Title>
+    <Title>Argent Pony Warcraft Client Dependency Injection</Title>
     <PackageTags>Warcraft;World-of-Warcraft;WoW;Blizzard;Dependency-Injection;DI</PackageTags>
```

Comment:
Would it make sense to replace the hyphens with spaces? Not sure exactly how NuGet's search behaviors will work if it includes dashes instead of spaces.

Reply:
That is a good question.  I can't find any prescriptive guidance on how NuGet package tags should be formatted.  I do see some with hyphens and others where the words are run together.  If I search NuGet.org for a tag with whitespace, like [dependency injection](https://www.nuget.org/packages?q=Tags%3A%22dependency+injection%22), I see packages with tags like `dependency` and `injection` and `DI`, but nothing with actual spaces in the tag.